public/pages/workflow_detail/workflow_inputs/processors_list.tsx

```diff
@@ -110,7 +110,7 @@ export function ProcessorsList(props: ProcessorsListProps) {
       return;
     }
 
-    if (dataSourceId) {
```

Comment:
can we check the mds feature flag to determine if mds is enabled instead of checking if dataSourceId is present? Checking the feature flag should be the best practice and less error-prone

Reply:
something like this - https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/public/pages/DetectorsList/containers/List/List.tsx#L147